aptos-move/framework/aptos-framework/doc/delegation_pool.md

```diff
@@ -4,89 +4,95 @@
-*  - the pending withdrawal exists at an OLC iff delegator owns some shares within the shares pool of that OLC.
-*
-* Example flow:
-* 1. A node operator creates a delegation pool by calling <code>initialize_delegation_pool</code> and sets
-* its commission fee to 0% (for simplicity). A stake pool is created with no initial stake and owned by
-* a resource account controlled by the delegation pool.
-* 2. Delegator A adds 100 stake which is converted to 100 shares into the active pool_u64
-* 3. Operator joins the validator set as the stake pool has now the minimum stake
-* 4. The stake pool earned rewards and now has 200 active stake. A's active shares are worth 200 coins as
-* the commission fee is 0%.
-* 5a. A requests <code>unlock</code> for 100 stake
-* 5b. Synchronization detects 200 - 100 active rewards which are entirely (0% commission) added to the active pool.
-* 5c. 100 coins = (100 * 100) / 200 = 50 shares are redeemed from the active pool and exchanged for 100 shares
-* into the pending_inactive one on A's behalf
-* 6. Delegator B adds 200 stake which is converted to (200 * 50) / 100 = 100 shares into the active pool
-* 7. The stake pool earned rewards and now has 600 active and 200 pending_inactive stake.
-* 8a. A requests <code>reactivate_stake</code> for 100 stake
-* 8b. Synchronization detects 600 - 300 active and 200 - 100 pending_inactive rewards which are both entirely
-* distributed to their corresponding pools
-* 8c. 100 coins = (100 * 100) / 200 = 50 shares are redeemed from the pending_inactive pool and exchanged for
-* (100 * 150) / 600 = 25 shares into the active one on A's behalf
-* 9. The lockup expires on the stake pool, inactivating the entire pending_inactive stake
-* 10a. B requests <code>unlock</code> for 100 stake
-* 10b. Synchronization detects no active or pending_inactive rewards, but 0 -> 100 inactive stake
-* on the stake pool, so it advances the observed lockup cycle and creates a pool_u64 for the new lockup,
-* hence allowing previous pending_inactive shares to be redeemed
-* 10c. 100 coins = (100 * 175) / 700 = 25 shares are redeemed from the active pool and exchanged for
-* 100 shares into the new pending_inactive one on B's behalf
-* 11. The stake pool earned rewards and now has some pending_inactive rewards.
-* 12a. A requests <code>withdraw</code> for its entire inactive stake
-* 12b. Synchronization detects no new inactive stake, but some pending_inactive rewards which are
-* distributed to the (2nd) pending_inactive pool
-* 12c. A's 50 shares = (50 * 100) / 50 = 100 coins are redeemed from the (1st) inactive pool and 100 stake
-* is transferred to A
-
-
+Allow multiple delegators to participate in the same stake pool in order to collect the minimum
+stake required to join the validator set. Delegators are rewarded out of the validator rewards
+proportionally to their stake and provided the same stake-management API as the stake pool owner.
+
```

Comment:
This is generated from the code.  You'll have to change the comments in the move code for the module.